### ISSUE TITLE
Cannot generate Singer Error Report.

### ISSUE THREAD

**Issue by gluustix** [stakira/OpenUtau#1322]
### Acknowledgement

- [X] I have read Getting-Started and FAQ

### 🐛 Describe the bug

I had an issue in the past which I tracked down to several of my singers becoming corrupted when I moved a large amount of files. I know how to fix them, I just need the help of a singer report to figure out which files are corrupt or missing, as there are a large amount and it's difficult to do by hand. However, I'm unable to generate a singer report? It just throws this. 
![image](https://github.com/user-attachments/assets/78b3ce8a-7c3b-4a25-8c21-539c3d35b359)
I'm not quite certain what's causing this or if this is my fault, or perhaps related to the corrupted singer files, but I would definitely like to figure this out so I can utilize the singer report to assist me in finding which files are a problem.

### Explains how to reproduce the bug

1. Open the program.
2. Open the singers menu
3. Attempt to generate singer report
4. Program throws error

### OS & Version

Windows 11 Home

### Logs

[log20241108.txt](https://github.com/user-attachments/files/17681587/log20241108.txt)

**Comment by BagelHero** [comment]
So sorry for the delay in getting to this, but I was able to reproduce this with a singer that had excess line breaks at the bottom of their oto.ini file. If you can locate any oto.ini files within this voicebank folder and manually check that they don't have any excess white space, this may help!

**Comment by gluustix** [comment]
No worries about the delay! I ended up busy and forgot about this all until now anyways. This solved it! I'd had to manually remove a couple excess entries that pointed to files I renamed in the past and I guess I didn't realize I left excess whitespace at the bottom when I did, but that wasn't even a possibility that crossed my mind in relation to the crashes. Odd that it does that but I could see why. Regardless, thanks a bunch! :)
